Automattic/vip-go-mu-plugins
# Adding Parse.ly 3.0

## Description

This PR adds the Parsely 3.0 version to make it available to sites. It is not the default version yet, so no changes should happen on the site unless the site has 3.0 as the default version:

```
add_filter(
	'wpvip_parsely_version',
	function() {
		return '3.0';
	} 
);

add_filter( 'wpvip_parsely_load_mu', '__return_true' );
```

## Changelog Description

### Adding Parse.ly 3.0

We added Parse.ly 3.0 plugin as a (non-default) option to sites.

## Checklist

Please make sure the items below have been covered before requesting a review:


## Steps to Test

1. Check out PR.
2. Make sure Parse.ly is enabled on your site.
3. Declare the constant to enable Parse.ly 3.0
